.github/workflows/ci.yml

```
@@ -142,7 +142,47 @@ jobs:
           private-github-key: ${{ secrets.PRIVATE_GITHUB_KEY }}
 
       - name: Run live dev env tests
-        run: docker exec tools_awx_1 /bin/bash -c "make live_test"
+        id: make-run
+        run: docker exec tools_awx_1 /bin/bash -c "GITHUB_ACTIONS=${GITHUB_ACTIONS} PYTEST_ADDOPTS='--cov=awx --cov-report=xml --junitxml=../../../../reports/junit.xml --no-cov-on-fail' make live_test"
```

The tests fail because we're attempting to `>>""` since another var hasn't been populated properly.
```suggestion
        run: docker exec tools_awx_1 /bin/bash -c "GITHUB_ACTIONS=${GITHUB_ACTIONS} GITHUB_OUTPUT=${GITHUB_OUTPUT} PYTEST_ADDOPTS='--cov=awx --cov-report=xml --junitxml=../../../../reports/junit.xml --no-cov-on-fail' make live_test"
```

In the future, you may also mirror this for `GITHUB_STEP_SUMMARY`, but it's not currently in use, so we can postpone that.